### PR TITLE
Update gitpython and certifi

### PR DESCRIPTION
**Pull request by Ch3LL**

### CODE REVIEW

**Review by garethgreenaway — DISMISSED** [review]

**Comment by Ch3LL** [comment]
re-run full all

**Review by twangboy — APPROVED** [review]

File: requirements/static/ci/common.in [hunk 1]
```diff
@@ -7,15 +7,17 @@ boto3>=1.16.0,<1.17.0; python_version < '3.6'
 boto3>=1.17.67; python_version >= '3.6'
 boto>=2.46.0
 cassandra-driver>=2.0
-certifi
+certifi>=2022.12.7; python_version >= "3.6"
+certifi<=2021.10.8; python_version <= "3.5"
```
Comment (s0undt3ch) [inline requirements/static/ci/common.in:11]: We don't run 3.5 anywhere these days...

**Review by s0undt3ch — APPROVED** [review]

File: requirements/static/ci/common.in (hunk 1 above)
Comment (Ch3LL) [reply requirements/static/ci/common.in:11]: We still support 3.5 in 3005 though. I know this is just for the tests here, but I figured might as well.

Comment (s0undt3ch) [reply requirements/static/ci/common.in:11]: Oh, wait, this is the freeze branch.

Comment (s0undt3ch) [reply requirements/static/ci/common.in:11]: We also have certifi in some PKG requirents.

Comment (s0undt3ch) [reply requirements/static/ci/common.in:11]: https://github.com/saltstack/salt/commit/7df5feb62be4d18fc5c053d656e35b354430cda8

Comment (Ch3LL) [reply requirements/static/ci/common.in:11]: Yeah, my pr is adding it to some of the pkg ones as well. Is there concern about this?

Comment (s0undt3ch) [reply requirements/static/ci/common.in:11]: No, no. Didn't saw them and didn't want them to get missed.

**Review by garethgreenaway — APPROVED** [review]

**Pull request merged**